src/pages/VenuePage/VenuePage.tsx

```diff
@@ -244,11 +244,7 @@ const VenuePage = () => {
   if (!user) {
     return (
       <WithNavigationBar>
-        <AuthenticationModal
-          show={true}
-          onHide={() => {}}
-          showAuth="register"
-        />
+        <AuthenticationModal show={true} onHide={() => {}} showAuth="initial" />
```

Comment:
The final goal is to get rid of this and use Login component instead.

Reply:
Instead of passing a noop function `onHide`, we should be making it an optional param in the component wanting it, + check that it exists before attempting to call it there. That'll avoid us needing to use workaround hacks like this.